python/ray/train/callbacks/logging.py

```diff
@@ -255,7 +204,7 @@ def start_training(self, logdir: str, config: Dict, **info):
         self.mlflow_util.log_params(params_to_log=config)
 
     def handle_result(self, results: List[Dict], **info):
-        result = results[self._workers_to_log]
+        result = results[0]
```

Comment:
This is 0 as it will always be one element, right?

Reply:
Yep let me add a comment. In this new pattern `handle_result` will always take in a list, but combined with `SingleWorkerResultsPreprocessor` it is expected to have length 1.